client/data/README.md

```diff
@@ -0,0 +1,136 @@
+# 🚀🚀 Blog Readme 
+
+> Ideally let's use an active branch for posting the blog contents...
+
+♻️ `Reduce, Reuse, Recycle` ♻️
+
+Hey there, welcome to the **Blog Branch**. Here, we'll post articles and content related to the 3Rs in Singapore. Hopefully, this will drive traffic and improve SEO rankings!
+
+***To create a post, here's how to do it:***
+
+
+## Selecting blog branch
+
+To limit access to development files and branches, let's select the blog branch by clicking on `main` on the upper-left corner of our code repository, as shown below:
+
+![How to switch branch](etc/switch_branch.png)
+
+Then, click on the `<yet undecided branch name>` branch.
+
+
+## Creating a post 📝
+
+1. Navigate to `recyclegowhere/client/data/blog/` folder and create an `.mdx` file by clicking on the top right **Add file > Create new file** button as shown below:
+
+    ![How to add file](etc/add_file.png)
+
+2. Type in your desired article name as the file name. Note that this will be the _slug_ (or URL parameter) for your blog post.
+
+    For example, if I'd like to see my URL as `https://recyclegowhere.org/uncle-sem-investigates-the-impacts-plastic-bottle`, my file name will be `uncle-sem-investigates-the-impacts-plastic-bottle.mdx` – as shown below:
+
+    ![How to name file](etc/rename_file.png)
+
+3. Key in the following code to the start of your `.mdx` file:
+
+    ```
+    ---
+    title: 'Test'
+    publishedAt: '2021-12-15'
+    summary: 'Test123'
+    author: 'Emmanuel Rayendra'
+    ---
+    ```
+
+    In `title`, type in your article name. Using my example above, my title will be `"Uncle Sem Investigates: The impacts of plastic bottles"`.
+
+    In `publishedAt`, please key in today's date in the following fashion: `YYYY-MM-DD`.
+
+    In `summary`, here you can state the summary of your article.
+
+    In `author`, kinda _self-explanatory_ yeah? 🙃
+
+***Once that's out of the way, you can start crafting your content! You could also refer to our past blog posts for examples...***
```

Comment:
How about adding a link to a dummy blogpost on GitHub for easy reference?

Reply:
`todo`